### ISSUE TITLE
Recaptcha v2 removes border from recaptcha button on focus/hover

### ISSUE THREAD

**Issue by annmathew111** [google/recaptcha#265]
Removing the border from recaptcha buttons causes an accessibility violation since the button state may not be clear to a low vision user.

**Comment by annm1991** [comment]
Removing the style ‘outline:none’ on .rc-button:hover, .rc-button:focus in styles_ltr.css is a potential fix for this issue.

**Comment by rowan-m** [comment]
Closing old issues that are not related to the PHP client code.
